libvcs4j-api/src/test/java/de/unibremen/informatik/st/libvcs4j/PositionTest.java

```diff
@@ -159,6 +159,24 @@ public void endOfLine() throws IOException {
 		assertThat(endOfLine.getTabSize()).isEqualTo(7);
 		assertThat(endOfLine.getOffset()).isEqualTo(20);
 		assertThat(endOfLine.getLineOffset()).isEqualTo(20);
+
```

Comment:
Please create a new test case. The name of the test could be `endOfLineWithTabs`.

Reply:
I pulled the test out of the old test case and made a new one for it.